pkg/flagset/flagset.go

```diff
@@ -159,6 +159,7 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        "mount-path",
+			Value:       "/var/tmp/ocis-settings",
```

Comment:
unrelated to this PR, but if we want to make this directories future proof, we must not rely on `/var/tmp/` to exist, and instead use the parent Reva config as Basepath. I'll open an issue for it.

Reply:
@kulmann when I start the settings service I got two folders that relate to the settings service:

```
drwx------  3 aunger  wheel    96B Jul  8 11:04 ocis-settings
drwx------  3 aunger  wheel    96B Jul  8 11:04 ocis-settings-store
```

Should we align these two? Source is here: https://github.com/owncloud/ocis-settings/blob/master/pkg/store/filesystem/store.go#L15

Reply:
As discussed, this happened by accident.... I wanted to align ocis-settings with ocis-accounts, which doesn't have a mount path, but only a data path. Forgot to change how the variable is used in the store implementation. 👀